tests/unit/s2n_psk_test.c

```diff
@@ -866,11 +866,12 @@ int main(int argc, char **argv)
 
             uint32_t offered_psks_size = 0;
             struct s2n_psk *test_psk = NULL;
+            EXPECT_SUCCESS(s2n_connection_set_psk_mode(conn, S2N_PSK_MODE_EXTERNAL));
             while(offered_psks_size < UINT16_MAX) {
                 EXPECT_OK(s2n_array_pushback(&conn->psk_params.psk_list, (void**) &test_psk));
                 EXPECT_NOT_NULL(test_psk);
 
-                EXPECT_OK(s2n_psk_init(test_psk, S2N_PSK_TYPE_RESUMPTION));
+                EXPECT_OK(s2n_psk_init(test_psk, S2N_PSK_TYPE_EXTERNAL));
```

Comment:
Just saw this in another PR, is the default changing, or something else causing the init value to be changed?

Reply:
I don't know about Madeleine's PR (maybe she's doing it preemptively?) but here it's just the "can't mix PSK types" restriction that this PR adds.